.github/workflows/apple-perf.yml

```diff
@@ -181,23 +153,23 @@ jobs:
         GITHUB_RUNNER=1 PYTHON_EXECUTABLE=python ${CONDA_RUN} --no-capture-output \
           .ci/scripts/setup-macos.sh "${BUILD_TOOL}"
 
-        if [[ ${{ matrix.delegate }} == "coreml" ]]; then
+        if [[ ${{ matrix.benchmark_configs.config }} == *"coreml"* ]]; then
```

Comment:
This line and the one on line 161 looks wrong, you probably mean `${{ matrix.config }}` here.  The failure looks cryptic, but it tells something wrong in the script https://github.com/pytorch/executorch/actions/runs/12387012961/job/34575783485#step:9:8209

Reply:
Oops, forget to fix this